crates/test/test-components/src/chain/traits/messages/ibc_transfer.rs

```diff
@@ -18,21 +21,20 @@ pub trait CanBuildIbcTokenTransferMessage<Counterparty>:
     + HasAmountType
     + HasMemoType
     + HasMessageType
-    + HasHeightType
-    + HasTimeoutType
     + HasChannelIdType<Counterparty>
     + HasPortIdType<Counterparty>
 where
-    Counterparty: HasAddressType,
+    Counterparty: HasAddressType + HasHeightType + HasTimeoutType,
 {
     async fn build_ibc_token_transfer_message(
```

Comment:
```suggestion
    async fn build_ibc_token_transfer_messages(
```

Reply:
Fixed in 8facaa352abf86d94919b46a6809c382a30ecf24.